docs/user-guide/cli-install-cli-next.md

```diff
@@ -0,0 +1,57 @@
+# Install the Zowe CLI @next version
+
+<Badge text="Technical Preview"/> The Zowe CLI @next release is a technical preview. Technical previews are for testing only and are not ready for production. Your feedback is valued and appreciated.
+
+Install the Zowe CLI `@next` version from the online registry. You can follow this procedure for a first-time installation, or to update a currently installed version.
+
+***Prerequisites***
+*  Meet the [software requirements for Zowe CLI](https://docs.zowe.org/stable/user-guide/systemrequirements.html#zowe-cli-requirements).
+*  Meet the [software requirements for each plug-in](https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html#software-requirements-for-zowe-cli-plug-ins).
+
+***Follow these steps:***
+
+1. To install or update the core CLI, open a command-line window:
+
+   ```
+   npm install -g @zowe/cli@next
+   ```
+   Zowe CLI is installed.
+
+
+2. (Optional) Check [npmjs.com](https://www.npmjs.com/) for any Zowe plug-ins that have an `@next` version available. If an `@next` version is available, you can install it: 
+
+    ```
+    zowe plugins install @zowe/<plugin-name>@next
+    ```
+    
+    If no `@next` version is available,  install the `@latest` version of the plug-in:
+
+    ```
+    zowe plugins install @zowe/<plugin-name>@latest
```

Comment:
You can keep this but I believe `@latest` is the default so it could just be `zowe plugins install @zowe/<plugin-name>@latest`